parm/config/gfs/config.base.emc.dyn

```diff
@@ -337,6 +337,8 @@ fi
 
 if [[ "${DOIAU_ENKF}" = "NO" ]]; then export IAUFHRS_ENKF="6"; fi
 
+export GSI_SOILANAL="NO"
```

Comment:
Can we make this a templated variable and set it to `NO` as the default?
That way, we can set this to `YES` in one of the existing tests and ensure this capability is tested regularly.
The test this could piggy back off on is [C96C48_hybatmDA.yaml](https://github.com/NOAA-EMC/global-workflow/blob/develop/ci/cases/pr/C96C48_hybatmDA.yaml)

Reply:
Rahul - I'm not familiar with the templating, but followed through the example for DOIAU. Is this all I need to do? 

In config.base.emc.dyn change the line highlighted above to:  

export GSI_SOILANAL=@GSI_SOILANAL@

Then in parm/config/gfs/yaml/defaults.yaml  I set the default to NO. 

For the test, should I add GSI_SOILANAL=YES to [C96C48_hybatmDA.yaml](https://github.com/NOAA-EMC/global-workflow/blob/develop/ci/cases/pr/C96C48_hybatmDA.yaml), or create a new test based on that one?